### PR TITLE
Added "become a mentor" call-to-action button

### PR DESCRIPTION
**Pull request by kdmarble**
Added "become a mentor" call-to-action button and some additional class styling

### CODE REVIEW

**Comment by phil-ociraptor** [comment]
Now that I see the buttons side by side, it looks off.

Can you try stacking the buttons one on top of the other? 

- Change the flex direction to column
- Add a bit of vertical margin between buttons
- Center the button text

Also, could you introduce a `button-secondary` class? We want the `button-primary` and `button-secondary` to have different styling, although for now, they can be pretty much the same.

**Comment by kdmarble** [comment]
Updated the layout of the buttons. Does look much better vertical.

Created a `button-secondary` class and used the same styling as `button-primary` for the time being

File: pages/index.js [hunk 1]
```diff
@@ -25,7 +25,12 @@ export default function Home() {
           Then you're in the perfect spot. We're a community of college students and industry mentors who are building this summer. We learn from each other. We meet on Thursdays for demos, presentations, and networking. The end goal is to make you similarly competitive to those who completed internships over the summer. 🚀
         </p>
 
-        <a className="button-primary" href="https://forms.gle/3N3CfqbH6YkM1DF67"> Join us! </a>
+        <div className="call-to-action">
+          <a className="button-primary" href="https://forms.gle/3N3CfqbH6YkM1DF67"> Join us! </a>
+
+          <a className="button-primary" href="https://forms.gle/ybJ7pdF5Q3axZbrbA"> Become a Mentor! </a>
```
Comment (phil-ociraptor) [inline pages/index.js:31]: supernit: can you change the class name to `button-secondary`?

File: pages/index.js [hunk 2]
```diff
@@ -222,6 +248,16 @@ export default function Home() {
...
+
+          .call-to-action {
+            width: 45%;
+            flex-direction: column;
+          }
+          
+          .call-to-action a {
+            margin: 1rem 0;
+          }
+          }
```
Comment (phil-ociraptor) [inline pages/index.js:260]: hmm, this appears to be an extra `}` - please remove

File: pages/index.js [hunk 3]
```diff
@@ -26,9 +26,9 @@ export default function Home() {
         </p>
 
         <div className="call-to-action">
-          <a className="button-primary" href="https://forms.gle/3N3CfqbH6YkM1DF67"> Join us! </a>
+          <a className="button-secondary" href="https://forms.gle/3N3CfqbH6YkM1DF67"> Join us! </a>
```
Comment (phil-ociraptor) [inline pages/index.js:29]: Should have been more clear. Put the `"button-primary"` on the "Join us!", but put the `"button-secondary"` on the "Become a Mentor!"

File: pages/index.js [hunk 4]
```diff
@@ -208,6 +225,15 @@ export default function Home() {
...
             font-size: 1.5rem;
+            text-align: center;
+        }
+
+        .button-secondary {
+          color: rgba(33, 33, 33, 1);
+          background-color: rgba(77,208,225, 1);
+          border-radius: 10px;
+          padding: 10px;
+          font-size: 1.5rem;
```
Comment (phil-ociraptor) [inline pages/index.js:236]: `.button-secondary` is missing a `text-align: center;` (which `button-primary` has)

**Comment by phil-ociraptor** [comment]
Add "Become a Mentor!" CTA Button

**Pull request merged**